main.go

```diff
@@ -207,12 +208,20 @@ func run(state overseer.State) {
 	}
 
 	ctx := context.TODO()
+	var detectorsOption engine.EngineOption
+
+	if len(*detectors) > 0 {
+		detectorsOption = engine.WithDetectors(!*noVerification, engine.Detectors(ctx, strings.Split(*detectors, ","))...)
+	} else {
+		detectorsOption = engine.WithDetectors(!*noVerification, engine.DefaultDetectors()...)
+	}
+
 	e := engine.Start(ctx,
 		engine.WithConcurrency(*concurrency),
 		engine.WithDecoders(decoders.DefaultDecoders()...),
-		engine.WithDetectors(!*noVerification, engine.DefaultDetectors()...),
 		engine.WithDetectors(!*noVerification, conf.Detectors...),
 		engine.WithFilterUnverified(*filterUnverified),
+		detectorsOption,
 	)
```

Comment:
Nice - so `--config config.yaml --detectors AWS` would use custom detectors and AWS detectors. Does that make sense, or should it be `--config config.yaml --detectors AWS,CustomRegex`?

Reply:
I think that's fine, given the CustomRegex is a bit special due to the fact we pass it via a config. It seems like maybe it should stand on its own? I don't have much of a preference though.

Reply:
oh, i just noticed your comment has a config either way.

Reply:
Yeah - in the future the config file might have more than one custom detector (or other config options), so being able to filter them by detector type makes sense to me?

Maybe I'm being overly cautious here but it's easier to relax restrictions than to add them so I slightly prefer `--detectors AWS,CustomRegex`.